app/src/main/java/com/sun/moviedb/utils/BindingUtils.kt

```diff
@@ -10,15 +10,45 @@ import com.bumptech.glide.load.resource.bitmap.RoundedCorners
 import com.bumptech.glide.request.RequestOptions
 import com.sun.moviedb.MovieApplication
 import com.sun.moviedb.R
+import com.sun.moviedb.data.entity.Movie
+import com.sun.moviedb.view.adapter.MovieHomeAdapter
 
 /**
  * Created by nguyenxuanhoi on 2019-08-17.
  * @author nguyen.xuan.hoi@sun-asterisk.com
  */
 object BindingUtils {
+    @BindingAdapter("bindTextCategory")
+    @JvmStatic
+    fun textCategory(textView: TextView, query: String) {
+        when (query) {
+            CategoryQuery.UP_COMING -> textView.text = MovieApplication.applicationContext!!.getString(R.string.title_up_coming)
+            CategoryQuery.POPULAR -> textView.text = MovieApplication.applicationContext!!.getString(R.string.title_popular)
+            CategoryQuery.NOW_PLAYING -> textView.text = MovieApplication.applicationContext!!.getString(R.string.title_now_playing)
+            CategoryQuery.TOP_RATE -> textView.text = MovieApplication.applicationContext!!.getString(R.string.title_top_rate)
+        }
+    }
+
+    @BindingAdapter("android:src")
```

Comment:
```suggestion
    @BindingAdapter("android:url")
```
đặt tên là src có thể trùng với attr default của android cung cấp
em có thể đặt tên ntn